template/public/app.js

```diff
@@ -1,34 +1,48 @@
-import moment from 'moment';
-import { uiModules } from 'ui/modules';
-import uiRoutes from 'ui/routes';
+import React from "react";
+import moment from "moment";
+import { uiModules } from "ui/modules";
+import chrome from "ui/chrome";
+import uiRoutes from "ui/routes";
+import { render, unmountComponentAtNode } from "react-dom";
 
-import 'ui/autoload/styles';
-import './less/main.less';
-import template from './templates/index.html';
+import "ui/autoload/styles";
+import "./less/main.less";
+import template from "./templates/index.html";
+import { Main } from "./components/main";
 
+const app = uiModules.get("apps/<%= name %>");
```

Comment:
This is partially a matter of personal preference, but can we add whitespace to break the code up into clearer chunks? I think I'd find this code easier to scan if we add newlines on line 14, 17, 35, and 36.